### PR TITLE
Upgrade to OpenVINO R1

### PR DESCRIPTION
**Pull request by SDxKeeper**

### CODE REVIEW

**Comment by jhou5** [comment]
CI failure found, can you pls. kindly look at it?
The following tests FAILED:
	  1 - test_xeon_dldt_ffmpeg_cent74_ffmpeg_1dns (Failed)
	  2 - test_xeon_dldt_ffmpeg_cent74_ffmpeg_flv_hevc (Failed)
	  3 - test_xeon_dldt_ffmpeg_cent74_ffmpeg_ivf_av1 (Failed)
	  5 - test_xeon_dldt_ffmpeg_cent74_ffmpeg_svt_av1 (Failed)
	  6 - test_xeon_dldt_ffmpeg_cent74_ffmpeg_svt_hevc (Failed)
	  8 - test_xeon_dldt_ffmpeg_cent74_ffmpeg_vpx (Failed)
	  9 - test_xeon_dldt_ffmpeg_cent74_ffmpeg_x264 (Failed)
	 10 - test_xeon_dldt_ffmpeg_cent74_ffmpeg_x265 (Failed)
Errors while running CTest

**Comment by fkhoshne** [comment]
The gstreamer-videoanalytics.m4 template will currently fail DLDT is updated to OpenVINO R1. There is a public release planned in a couple of weeks to update the video analytic plugins to support R1, at which time the dldt.m4 template can be updated to R1

**Comment by SDxKeeper** [comment]
@linxie47 can you please take a look on ffmpeg tests failures?

**Comment by linxie47** [comment]
@SDxKeeper I found it was caused by incorrect environment
**LD_LIBRARY_PATH**=:/opt/intel/dldt/inference-engine/lib:/opt/intel/dldt/inference-engine/external/tbb/lib:**/opt/intel/dldt/inference-engine/lib/centos_7.4/intel64**

The actual path is **/opt/intel/dldt/inference-engine/lib/intel64**

File: template/dldt-ie.m4 [hunk 1]
```diff
@@ -4,7 +4,7 @@ ARG DLDT_REPO=https://github.com/opencv/dldt.git
 ARG DLDT_C_API_1=https://raw.githubusercontent.com/VCDP/FFmpeg-patch/master/thirdparty/0001-Add-inference-engine-C-API.patch
 ARG DLDT_C_API_2=https://raw.githubusercontent.com/VCDP/FFmpeg-patch/master/thirdparty/0002-Change-to-match-image-with-separate-planes.patch
 ARG DLDT_C_API_3=https://raw.githubusercontent.com/VCDP/FFmpeg-patch/master/thirdparty/0003-Refine-IE-C-API.patch
-
+ARG DLDT_C_API_4=https://gist.githubusercontent.com/SDxKeeper/cfdd8595a6555846bd08b020fe8d3823/raw/be74dd0afb4e5efa168fa89e56b62a1de18951dc/0001-Enable-visibility-of-C-API.patch
```
Comment (linxie47) [inline template/dldt-ie.m4]: Could you please use patch
https://raw.githubusercontent.com/VCDP/FFmpeg-patch/master/thirdparty/0004-Fix-code-style-and-symbols-visibility-for-2019R1.patch to replace this? 
Thanks a lot!!

Comment (SDxKeeper) [reply template/dldt-ie.m4]: Sure, replaced

**Pull request merged**